R/busca_multi.R

```diff
@@ -0,0 +1,20 @@
+#' @title Busca por CEP
+#' @description Busca usando multiplos CEP (search for information using multiple postal code).
+#' @importFrom magrittr %>%
+#' @importFrom magrittr %<>%
+#' @importFrom dplyr add_row
+#' @param lista_ceps lista de CEPs (postal code list).
+#' @param token Token de autorização. Veja <http://cepaberto.com/users/register>.
+#' @export
+
+busca_multi <- function( lista_ceps = c("01001000", "01001001") , token = NULL ) {
+  CEPS <- parse_api(NA) %>% na.exclude()
+  lista_ceps %<>%  as.vector()
```

Comment:
Is `as.vector()` necessary here? I get:

```r
> lista_ceps = c("01001000", "01001001")
> identical(lista_ceps, as.vector(lista_ceps))
[1] TRUE
```

Reply:
I was having problems using data from tibbles directly into the function - e.g. I have a dataset with names of stores and their CEP and want an approximate latitude and longitude, so ideally my code would just include dataset$CEP %>% busca_multi() -, and this solved the problem by making sure that the user can either use data directly from a new vector or by choosing a column of a tibble or data frame.

Reply:
Sure, I understand. I think it's fine.